Automattic/woocommerce-payments
# Reporting: Add tooltips to each Payment activity widget tile

Fixes #8511 

#### Changes proposed in this Pull Request

Add tooltip with the requisite text to `Total payment volume`, `Charges` and `Fees` tile. 

#### Testing instructions
* Check out this branch
* Browse to `Payments > Overview`
* Within the `Your payment activity` widget, click on the info tooltips for `Total payment volume`, `Charges` and `Fees` tiles.
* The tooltips should match the screenshots or the design in Figma.
* Please also check in mobile screen or proportions. The tooltip modals should not be clipped. 

#### Screenshots

<a href="https://github.com/Automattic/woocommerce-payments/assets/4162931/6fc9d34c-e34f-4e38-a11c-362009ffc224"><img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/6fc9d34c-e34f-4e38-a11c-362009ffc224+" width="550px" /></a> 

<a href="https://github.com/Automattic/woocommerce-payments/assets/4162931/a96d1a91-8e7a-484e-88fe-5142d6baedd6"><img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/a96d1a91-8e7a-484e-88fe-5142d6baedd6+" width="550px" /></a>

<a href="https://github.com/Automattic/woocommerce-payments/assets/4162931/3a1a90d7-dacd-400d-a5af-5213455e6dae"><img src="https://github.com/Automattic/woocommerce-payments/assets/4162931/3a1a90d7-dacd-400d-a5af-5213455e6dae+" width="550px" /></a>

* Figma design : bIfbBgyBPRzvXEl7dEg3cv-fi-1373_44283 
-------------------


**Post merge**
